src/Pmi/Review/Review.php

```diff
@@ -0,0 +1,178 @@
+                        }
+                    } else {
+                        $participants[$participantId]['orderCount']++;
+                    }
+                    break;
+                case 'measurement':
+                    if (is_null($participants[$participantId]['physicalMeasurement'])) {
+                        $participants[$participantId]['physicalMeasurement'] = $row;
+                        $participants[$participantId]['physicalMeasurementCount'] = 1;
+                        // Get physical measurements status
+                        foreach (self::$measurementsStatus as $field => $status) {
+                            if ($row[$field]) {
+                                $participants[$participantId]['physicalMeasurementStatus'] = $this->getEvaluationStatus($row, $status);
+                            }
+                        }
+                    } else {
+                        $participants[$participantId]['physicalMeasurementCount']++;
+                    }
+                    break;
+            }
+        }
+
+        return $participants;
+    }
+
+    protected function getTodayOrderRows($db, $today)
+    {
+        $ordersQuery = 'SELECT o.*, ' .
+            'oh.type as h_type, ' .
+            's.name as created_site_name, ' .
+            'sc.name as collected_site_name, ' .
+            'sp.name as processed_site_name, ' .
+            'sf.name as finalized_site_name ' .
+            'FROM orders o ' .
+            'LEFT JOIN orders_history oh ON o.history_id = oh.id ' .
+            'LEFT JOIN sites s ON s.site_id = o.site ' .
+            'LEFT JOIN sites sc ON sc.site_id = o.collected_site ' .
+            'LEFT JOIN sites sp ON sp.site_id = o.processed_site ' .
+            'LEFT JOIN sites sf ON sf.site_id = o.finalized_site ' .
+            'WHERE ' .
+            '(o.created_ts >= :today OR o.collected_ts >= :today OR o.processed_ts >= :today OR o.finalized_ts >= :today OR oh.created_ts >= :today) ' .
+            'ORDER BY o.created_ts DESC';
+
+        return $db->fetchAll($ordersQuery, [
+            'today' => $today
+        ]);
+    }
+
+    public function getTodayOrders($db, $today)
+    {
+        $orders = [];
+        foreach ($this->getTodayOrderRows($db, $today) as $row) {
+            // Get order status
+            foreach (self::$orderStatus as $field => $status) {
+                if ($row[$field]) {
+                    $row['orderStatus'] = $this->getOrderStatus($row, $status);
+                }
+            }
+            // Get number of finalized samples
+            if ($row['finalized_samples'] && ($samples = json_decode($row['finalized_samples'])) && is_array($samples)) {
+                $row['finalizedSamples'] = count($samples);
+            }
+            $orders[] = $row;
+        }
+
+        return $orders;
+    }
+
+    public function getOrderStatus($row, $status)
+    {
+        $order = new Order;
```

Comment:
If `$order` is only used for accessing static constants, just access them directly (e.g. `Order::ORDER_CANCEL`)